index.html

```diff
@@ -14,23 +14,157 @@
+                <button class="button" type="button">See Project</button>
+            </div>
+        </section>
+        <section class="card card-3">
+            <div class="card-img-3 card-img"></div>
+            <div class="card-text">
+                <h2>Tonic</h2>
+                <div class="title-role">
+                    <ul>
+                        <li>CANOPY</li>
+                        <li class="role"><i class="fas fa-circle"></i></li>
+                        <li class="role">Back End Dev</li>
+                        <li class="role"><i class="fas fa-circle"></i></li>
+                        <li class="role">2015</li>
+                    </ul>
+                </div>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <div class="languages">
+                    <ul>
+                        <li><a href="#">html</a></li>
+                        <li><a href="#">css</a></li>
+                        <li><a href="#">javascript</a></li>
+                    </ul>
+                </div>
+                <button class="button" type="button">See Project</button>
+            </div>
+        </section>
+        <section class="card card-4">
+            <div class="card-img-4 card-img"></div>
+            <div class="card-text">
+                <h2>Multi-Post Stories</h2>
+                <div class="title-role">
+                    <ul>
+                        <li>CANOPY</li>
+                        <li class="role"><i class="fas fa-circle"></i></li>
+                        <li class="role">Back End Dev</li>
+                        <li class="role"><i class="fas fa-circle"></i></li>
+                        <li class="role">2015</li>
+                    </ul>
+                </div>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <div class="languages">
+                    <ul>
+                        <li><a href="#">html</a></li>
+                        <li><a href="#">css</a></li>
+                        <li><a href="#">javascript</a></li>
+                    </ul>
+                </div>
+                <button class="button" type="button">See Project</button>
+            </div>
+        </section>
+    </section>
+    <section class="about-me headline-container headline">
+        <div class="aboutme-text">
+            <h1>About Myself</h1>
+            <p>Hello I’m a software developer! I can help you build a product ,
                 feature or website Look through some of my work and experience!
-                If you like what you see and have a project you need coded,
-                don’t hestiate to contact me.</p>
+                If you like what you see and have a project you need coded, don’t
+                hestiate to contact me.</p>
             <a href="#">Let's connect</a>
             <ul class="social-icons">
                 <li>
```

Comment:
- [ ] It seems that you are having an `overflow-x` that is not giving a good UX, also the about me section is not taking the full width,
- [ ] Also the languages (right) section seems to take a big width compared to the about myself (left) in the Figma design, but n your design actually they have equal width, see the difference below 👇 

- Figma design
![image](https://user-images.githubusercontent.com/55465133/130154299-b20a66b1-acd3-4294-a039-bd8ebd77f27a.png)

- Your design
![image](https://user-images.githubusercontent.com/55465133/130154353-ff08f24c-8371-46c6-8084-39e4ff8f427a.png)

Can we please fix this 😉 ?